src/lib/stores/results.svelte.ts

```diff
@@ -100,18 +103,17 @@ export function withResultsStore() {
 				.add(newTotalWithdrawFee)
 				.add(newTotalManagementFee)
 				.add(newTotalSuccessFee)
-
-			gd.push({
-				date: new Date(i),
-				totalInvested: newTotalInvested.toNumber(),
-				totalDeposited: newTotalDeposited.toNumber(),
-				totalWithdrawn: newTotalWithdrawn.toNumber(),
-				totalFees: newTotalFees.toNumber(),
-				totalManagementFee: newTotalManagementFee.toNumber(),
-				totalSuccessFee: newTotalSuccessFee.toNumber(),
-				totalDepositFee: newTotalDepositFee.toNumber(),
-				totalWithdrawFee: newTotalWithdrawFee.toNumber(),
-			})
+			if (i.getDate() % 30 === 0) {
```

Comment:
This is probable better - the values will be store for every first day of the month.
```suggestion
			if (i.getDate()  === 0) {
```